CorsixTH/Lua/app.lua

```diff
@@ -313,6 +313,39 @@ function App:init()
   return true
 end
 
+---
+-- Initialises a Lua DBGp client and
+-- connects it to a DBGp server so that
+-- it can debug lua scripts at runtime.
+-- @return error_message
```

Comment:
LDocGen was custom made by Corsix.  I'm quite ok with moving to some more standard format as long as all the tooling is updated at the same time.  Such a project is out of scope for this pull request though.